src/js/Expression.js

```diff
@@ -0,0 +1,46 @@
+import Calculator from "./Calculator.js"
+
+export default class Expression {
+
+    calculator;
+    operands;
+    operator;
+
+    constructor() {
+        this.calculator= new Calculator();
+        this.clear();
+    }
+
+    getOperand(operand){
+        console.log(`${this.operands} | ${this.operator} | ${this.result}`);
+
+        this.operands.push(parseInt(operand))
+        return this.getExpression();
+    }
+
+    getOperator(operator){
+        console.log(`${this.operands} | ${this.operator} | ${this.result}`);
+        if(this.calculator.isOperator(operator)){
+            this.operator = operator;
+            return this.getExpression();
+        }
```

Comment:
얼리 리턴 하면 함수가 더 빨라진다(?)고 들었는데, 아래와 같은 코드는 어떠세용ㅎㅎ?
```suggestion
        if(!this.calculator.isOperator(operator))return
         this.operator = operator;
         return this.getExpression();
```